tests/ballerina-test/src/test/resources/test-src/nativeimpl/functions/compression.bal

```diff
@@ -0,0 +1,17 @@
+import ballerina.compression;
+
+function testUnzipFile(string dirPath, string destDir) {
+    compression:unzipFile(dirPath, destDir);
+}
+
+function testZipFile(string dirPath, string destDir) {
+    compression:zipFile(dirPath, destDir);
+}
+
+function testUnzipBytes(blob content, string destDir) {
+    compression:unzipBytes(content, destDir);
+}
+
+function testZipToBytes(string dirPath) (blob) {
+    return compression:zipToBytes(dirPath);
+}
```

Comment:
Add test cases for passing null as the content/dirPath/destPath

Reply:
Added test cases to cover all scenarios when null is passed. Fixed with 5ea59f6

Reply:
We may need to update the test cases. Please see my below comment.